hack/non-olm-install/run-e2e-kuttl-test.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+export NON_OLM="true"
+
+WORKING_DIR=../..
+sequential_suite=$WORKING_DIR/test/openshift/e2e/sequential/
+parallel_suite=$WORKING_DIR/test/openshift/e2e/parallel/
+
+# Check if any argument is provided
+if [ $# -eq 0 ]; then
+    echo "No directory name provided, please add directory names as arguments while running the script"
```

Comment:
What directory names are we looking for here? If no directory names are provided, we should execute all sequential and parallel tests by default. WDYT?

Reply:
Yes, that is the behaviour

Reply:
Oh, I will update the statement to - `no directory provided, this will run all the tests in the suite`